Upstream packages a number of tools which define translations for gettext to
process. The glade files don't appear to actually have translation marks which
causes the build to complain and, in the case of packaging, fail.

--- po/POTFILES.skip.orig	2021-02-20 21:51:50.867875091 -0800
+++ po/POTFILES.skip	2021-02-20 21:51:57.689896618 -0800
@@ -4,3 +4,151 @@
 plugins/pltbrowser/support.c
 plugins/gtkui/support.c
+tools/glade/examples/editor/editor.glade
+tools/glade/examples/editor/src/interface.c
+tools/glade/examples/editor/src/support.c
+tools/glade/glade-2.desktop.in
+tools/glade/glade/editor.c
+tools/glade/glade/gbwidget.c
+tools/glade/glade/gbwidgets/gbaboutdialog.c
+tools/glade/glade/gbwidgets/gbaccellabel.c
+tools/glade/glade/gbwidgets/gbalignment.c
+tools/glade/glade/gbwidgets/gbarrow.c
+tools/glade/glade/gbwidgets/gbaspectframe.c
+tools/glade/glade/gbwidgets/gbbutton.c
+tools/glade/glade/gbwidgets/gbcalendar.c
+tools/glade/glade/gbwidgets/gbcellview.c
+tools/glade/glade/gbwidgets/gbcheckbutton.c
+tools/glade/glade/gbwidgets/gbcheckmenuitem.c
+tools/glade/glade/gbwidgets/gbclist.c
+tools/glade/glade/gbwidgets/gbcolorbutton.c
+tools/glade/glade/gbwidgets/gbcolorselection.c
+tools/glade/glade/gbwidgets/gbcolorselectiondialog.c
+tools/glade/glade/gbwidgets/gbcombo.c
+tools/glade/glade/gbwidgets/gbcombobox.c
+tools/glade/glade/gbwidgets/gbcomboboxentry.c
+tools/glade/glade/gbwidgets/gbctree.c
+tools/glade/glade/gbwidgets/gbcurve.c
+tools/glade/glade/gbwidgets/gbcustom.c
+tools/glade/glade/gbwidgets/gbdialog.c
+tools/glade/glade/gbwidgets/gbdrawingarea.c
+tools/glade/glade/gbwidgets/gbentry.c
+tools/glade/glade/gbwidgets/gbeventbox.c
+tools/glade/glade/gbwidgets/gbexpander.c
+tools/glade/glade/gbwidgets/gbfilechooserbutton.c
+tools/glade/glade/gbwidgets/gbfilechooserdialog.c
+tools/glade/glade/gbwidgets/gbfilechooserwidget.c
+tools/glade/glade/gbwidgets/gbfileselection.c
+tools/glade/glade/gbwidgets/gbfixed.c
+tools/glade/glade/gbwidgets/gbfontbutton.c
+tools/glade/glade/gbwidgets/gbfontselection.c
+tools/glade/glade/gbwidgets/gbfontselectiondialog.c
+tools/glade/glade/gbwidgets/gbframe.c
+tools/glade/glade/gbwidgets/gbgammacurve.c
+tools/glade/glade/gbwidgets/gbhandlebox.c
+tools/glade/glade/gbwidgets/gbhbox.c
+tools/glade/glade/gbwidgets/gbhbuttonbox.c
+tools/glade/glade/gbwidgets/gbhpaned.c
+tools/glade/glade/gbwidgets/gbhruler.c
+tools/glade/glade/gbwidgets/gbhscale.c
+tools/glade/glade/gbwidgets/gbhscrollbar.c
+tools/glade/glade/gbwidgets/gbhseparator.c
+tools/glade/glade/gbwidgets/gbiconview.c
+tools/glade/glade/gbwidgets/gbimage.c
+tools/glade/glade/gbwidgets/gbimagemenuitem.c
+tools/glade/glade/gbwidgets/gbinputdialog.c
+tools/glade/glade/gbwidgets/gblabel.c
+tools/glade/glade/gbwidgets/gblayout.c
+tools/glade/glade/gbwidgets/gblist.c
+tools/glade/glade/gbwidgets/gblistitem.c
+tools/glade/glade/gbwidgets/gbmenu.c
+tools/glade/glade/gbwidgets/gbmenubar.c
+tools/glade/glade/gbwidgets/gbmenuitem.c
+tools/glade/glade/gbwidgets/gbmenutoolbutton.c
+tools/glade/glade/gbwidgets/gbnotebook.c
+tools/glade/glade/gbwidgets/gboptionmenu.c
+tools/glade/glade/gbwidgets/gbpreview.c
+tools/glade/glade/gbwidgets/gbprogressbar.c
+tools/glade/glade/gbwidgets/gbradiobutton.c
+tools/glade/glade/gbwidgets/gbradiomenuitem.c
+tools/glade/glade/gbwidgets/gbradiotoolbutton.c
+tools/glade/glade/gbwidgets/gbscrolledwindow.c
+tools/glade/glade/gbwidgets/gbseparatormenuitem.c
+tools/glade/glade/gbwidgets/gbseparatortoolitem.c
+tools/glade/glade/gbwidgets/gbspinbutton.c
+tools/glade/glade/gbwidgets/gbstatusbar.c
+tools/glade/glade/gbwidgets/gbtable.c
+tools/glade/glade/gbwidgets/gbtextview.c
+tools/glade/glade/gbwidgets/gbtogglebutton.c
+tools/glade/glade/gbwidgets/gbtoggletoolbutton.c
+tools/glade/glade/gbwidgets/gbtoolbar.c
+tools/glade/glade/gbwidgets/gbtoolbutton.c
+tools/glade/glade/gbwidgets/gbtoolitem.c
+tools/glade/glade/gbwidgets/gbtreeview.c
+tools/glade/glade/gbwidgets/gbvbox.c
+tools/glade/glade/gbwidgets/gbvbuttonbox.c
+tools/glade/glade/gbwidgets/gbviewport.c
+tools/glade/glade/gbwidgets/gbvpaned.c
+tools/glade/glade/gbwidgets/gbvruler.c
+tools/glade/glade/gbwidgets/gbvscale.c
+tools/glade/glade/gbwidgets/gbvscrollbar.c
+tools/glade/glade/gbwidgets/gbvseparator.c
+tools/glade/glade/gbwidgets/gbwindow.c
+tools/glade/glade/glade.c
+tools/glade/glade/glade_atk.c
+tools/glade/glade/glade_clipboard.c
+tools/glade/glade/glade_gnome.c
+tools/glade/glade/glade_gnomelib.c
+tools/glade/glade/glade_gtk12lib.c
+tools/glade/glade/glade_keys_dialog.c
+tools/glade/glade/glade_menu_editor.c
+tools/glade/glade/glade_palette.c
+tools/glade/glade/glade_project.c
+tools/glade/glade/glade_project_options.c
+tools/glade/glade/glade_project_window.c
+tools/glade/glade/gnome-db/gnomedbcombo.c
+tools/glade/glade/gnome-db/gnomedbconnectprop.c
+tools/glade/glade/gnome-db/gnomedbdsnconfig.c
+tools/glade/glade/gnome-db/gnomedbdsndruid.c
+tools/glade/glade/gnome-db/gnomedbeditor.c
+tools/glade/glade/gnome-db/gnomedberror.c
+tools/glade/glade/gnome-db/gnomedberrordlg.c
+tools/glade/glade/gnome-db/gnomedbform.c
+tools/glade/glade/gnome-db/gnomedbgraybar.c
+tools/glade/glade/gnome-db/gnomedbgrid.c
+tools/glade/glade/gnome-db/gnomedblogin.c
+tools/glade/glade/gnome-db/gnomedblogindlg.c
+tools/glade/glade/gnome-db/gnomedbprovidersel.c
+tools/glade/glade/gnome-db/gnomedbsourcesel.c
+tools/glade/glade/gnome-db/gnomedbtableeditor.c
+tools/glade/glade/gnome/bonobodock.c
+tools/glade/glade/gnome/bonobodockitem.c
+tools/glade/glade/gnome/gnomeabout.c
+tools/glade/glade/gnome/gnomeapp.c
+tools/glade/glade/gnome/gnomeappbar.c
+tools/glade/glade/gnome/gnomecanvas.c
+tools/glade/glade/gnome/gnomecolorpicker.c
+tools/glade/glade/gnome/gnomecontrol.c
+tools/glade/glade/gnome/gnomedateedit.c
+tools/glade/glade/gnome/gnomedialog.c
+tools/glade/glade/gnome/gnomedruid.c
+tools/glade/glade/gnome/gnomedruidpageedge.c
+tools/glade/glade/gnome/gnomedruidpagestandard.c
+tools/glade/glade/gnome/gnomeentry.c
+tools/glade/glade/gnome/gnomefileentry.c
+tools/glade/glade/gnome/gnomefontpicker.c
+tools/glade/glade/gnome/gnomehref.c
+tools/glade/glade/gnome/gnomeiconentry.c
+tools/glade/glade/gnome/gnomeiconlist.c
+tools/glade/glade/gnome/gnomeiconselection.c
+tools/glade/glade/gnome/gnomemessagebox.c
+tools/glade/glade/gnome/gnomepixmap.c
+tools/glade/glade/gnome/gnomepixmapentry.c
+tools/glade/glade/gnome/gnomepropertybox.c
+tools/glade/glade/main.c
+tools/glade/glade/palette.c
+tools/glade/glade/property.c
+tools/glade/glade/save.c
+tools/glade/glade/source.c
+tools/glade/glade/tree.c
+tools/glade/glade/utils.c
